source/django/issues-and-help.txt

```diff
@@ -0,0 +1,40 @@
+.. _django-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: suggestion, bug, github, django
+
+We're glad to have such a vibrant community of users of the {+django-odm+}. We
+recommend seeking support for general
+questions through the :community-forum:`MongoDB Community Forums. <tag/python/>`
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the {+django-odm+},
+please open an issue in JIRA, our issue-management tool, using the
+following steps:
+
+1. `Create a JIRA account <https://jira.mongodb.org>`__.
+#. Navigate to `the Python Integrations project <https://jira.mongodb.org/projects/INTPYTHON/>`__.
+#. Click :guilabel:`Create Issue`. Please provide as much information as possible about the
```

Comment:
```suggestion
#. Click :guilabel:`Create` to create an issue. Please provide as much information as possible about the
```
Q: Are you referring to the green 'Create' button at the top of the page? Should edit if so

Reply:
Looks like that one does the same thing, but this is referring to the `+ Create Issue` at the bottom of the list of issues I think